source/utils/types.ts

```diff
@@ -117,7 +117,7 @@ export interface Delays {
 	request?: number;
 }
 
-export type Headers = Record<string, string | string[] | undefined>;
+export type Headers = Record<string, string | string[] | number | undefined>;
```

Comment:
I don't think we should support `number` here. Headers are strings, so I think it's better to force users to explicitly coerce/cast to a string themselves.

Reply:
This is referenced from the test for `content-length: 0`. Should I alter the test?

Reply:
Yes